Fix: when using custom table the choice list was still populated from core tags table

jaanusnurmoja opened the pull request:
tags.js uses now options=com_fabrik url (instead com_tags) but it shouldn't change anything as the default table is still #__tags

pollen8 commented on plugins/fabrik_element/tags/tags.php:
```diff
@@ -326,10 +328,26 @@ protected function buildQuery($data = array(), $incWhere = true, $opts = array()
 		return $query;
 	}
 
+	public function allTagsJSON()
```
could you add a PHPdoc block comment to this function please?

pollen8 commented on plugins/fabrik_element/tags/tags.php:
```diff
@@ -60,7 +60,10 @@ public function elementJavascript($repeatCounter)
+		$opts->formid = $this->getFormModel()->id;
+		$opts->listid = $this->getListModel()->getId();
+		$opts->url = $this->tagListURL();
 		return array('FbTags', $id, $opts);
```
codestyle: best to add a new line before the return statement for

pollen8 commented on plugins/fabrik_element/tags/tags.php:
```diff
@@ -60,7 +60,10 @@ public function elementJavascript($repeatCounter)
+		$opts->allTags = $this->allTagsJSON();
+		$opts->formid = $this->getFormModel()->id;
+		$opts->listid = $this->getListModel()->getId();
+		$opts->url = $this->tagListURL();
```
code style: best to add a line return before the return statement

jaanusnurmoja commented:
@pollen8 Done with some cleanup

jaanusnurmoja commented:
Reason?

jaanusnurmoja commented:
@cheesegrits OK,  changed the fields.xml - removed tags_dbname_desc element and moved the description to the tags_dbname, but also added more information to the label to be seen immediately. + some more works.

jaanusnurmoja commented:
P.S. Please, can anyone make a minified js as tags.js is also changed
